offchain-modules/packages/x/src/handlers/ckb.ts

```diff
@@ -743,9 +843,63 @@ export class CkbHandler {
     }
   }
 
+  async parseLockTx(tx: Transaction): Promise<CkbLockCellData | null> {
```

Comment:
Logic should be  safe for recognize lockTx, make sure user cannot generate a fake lockTx but pass the `fn parseLockTx` logic. We should verify:

1. inputs only contain one lockscript && the lockscirpt !== committeeMultisigLockscript
2. outputs[0].lockscirpt === committeeMultisigLockscript 
2.1 if utputs[0].typescript && outputs[0].typescript.codeHash === sudtTypescript.codeHash && outputs[0].typescript.hashType === sudtTypescript.hashType, then assetIdent = Hash(outputs[0].typescript), assetLockedAmount === outputs[0].data, bridgeFee = outputs[0].capacity
2.2 if utputs[0].typescript === null, then assetIdent = 0x000...000, assetLockedAmount = outputs[0].capacity - bridgeFee, bridgeFee = config.bridgeFee
3.  outputs[1..].lockscirpt !== committeeMultisigLockscript 

then parse lockMemo from witness.

Reply:
fixed:
1. all lockscript of inputs are the same and not == committeeMultisigLockscript
2. lockscript of the first output === committeeMultisigLockscript, 
  - if (!typescript):  ckb,  get amount from capacity
  - else:  sudt, get amount from data
3. only the first output lockscript can be committeeMultisigLockscript